src/containers/TemplateOptions/TemplateOptions.tsx

```
@@ -52,7 +76,19 @@ export const TemplateOptions = ({
     QUICK_REPLY: 'Quick Reply',
   };
   const options = ['Static', 'Dynamic'];
+  const [forms, setForms] = useState<any>([]);
+  const [form, setForm] = useState<any>(null);
   const { urlType, sampleSuffix } = dynamicUrlParams;
+
+  const {} = useQuery(LIST_WHATSAPP_FORMS, {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)